### PR TITLE
Feature/notes form

### LINKED ISSUE #11: As a student, if I haven't completed a module, I'll see a message stating that there is no feedback yet for that module.

### LINKED ISSUE #21: Create form for adding personal notes

### PR DESCRIPTION
**Pull request by piknikki**
### What does this PR do?
* I made a super basic landing for when the student first logs in
* I started implementing the style guide from Turing (called Savile)
* I created a form and then styled it
* It's not too terrible (doesn't make me cry)
* but it still needs to hook up to the fetch, so we'll do that in a new ticket once Jessica's biz is merged.
* I also put in a note for when there's no module data.
* I did some css to the module indicator at the top of the feedback window but I am starting to doubt whether we need this now.
* Then I did a bunch of other stuff that's been bugging me.

### Anything specific the reviewer should look at?
* Check it out in your UI. It's not bad on the eyes. 

Closes #21
Closes #11

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -5,9 +5,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
-<!--  google fonts  -->
+<!--  Savile Design System and google fonts  -->
+    <link rel="stylesheet" href="https://savile.turing.edu/css/v1/1.3.css">
```
Comment (knees4bees) [inline public/index.html:9]: Woot woot!!!

File: src/components/StudentFeedback.vue [hunk 2]
```diff
@@ -23,9 +23,38 @@
         </p>
       </div>
     </div>
+    <div v-else>
+      <h1 class="s-h1">There is nothing for this module yet. Stay tuned.</h1>
+    </div>
```
Comment (knees4bees) [inline src/components/StudentFeedback.vue:28]: Great addition with the `v-if` and `v-else` to conditionally render this error-ish message

File: src/components/Category.vue [hunk 3]
```diff
@@ -1,9 +1,12 @@
 <template>
-  <div class="feedback-container__category">
-    <p class="feedback-container__category--label">{{ projectFeedback.rubric_category_name }} </p>
+  <div class="feedback-container__category s-content">
```
Comment (knees4bees) [inline src/components/Category.vue:2]: I think the `s-content` here is referencing something from the Savile style sheet. Cool!

File: src/components/Header.vue [hunk 4]
```diff
@@ -27,7 +27,7 @@
 export default {
   data () {
     return {
-      date: '05/15/2015'
+      date: new Date().toLocaleDateString()
```
Comment (knees4bees) [inline src/components/Header.vue:30]: Boom

File: src/components/StudentFeedback.vue [hunk 5]
```diff
@@ -23,9 +23,38 @@
...
+                      required
+            >
+            </textarea>
+          </div>
+          <div class="buttons form-container__item--buttons">
+            <button class="s-button-primary-inverse reset" type="reset">Reset</button>
+            <button class="s-button-primary add">Add</button>
+          </div>
+        </form>
+      </div>
```
Comment (knees4bees) [inline src/components/StudentFeedback.vue:57]: Heck yeah, Nikki! Strong work here!! This note form is so so functional and I love it

File: src/styles/styles.scss [hunk 6]
```diff
@@ -1,7 +1,11 @@
+@font-face {
+  font-family: Helvetica;
+  src: url(../assets/fonts/Helvetica_Neue_LT_Std-W55_Normal.otf);
```
Comment (knees4bees) [inline src/styles/styles.scss:3]: I love Helvetica. Nice that you were able to grab it!

**Review by knees4bees — APPROVED** [review]
Nikki this is some fantastic work!

ETA: The build is failing only because of Cypress it looks like (so we can merge anyway, or maybe merge my branch first and then do some rebasing as we go ... we'll figure it out)

File: src/styles/styles.scss (hunk 6 above)
Comment (piknikki) [reply src/styles/styles.scss:3]: Yeah, it's part of Savile!!

File: src/components/StudentFeedback.vue [hunk 7]
```diff
@@ -23,9 +23,38 @@
...
+    </div>
 
     <section class="notes-container">
-      <h2>Notes section goes here</h2>
+      <p class="feedback-container__category--label">
+        <span class="s-h2">Student Notes</span>
+      </p>
+      <div class="current-notes">
+        <ul>
+          <li v-for="(note, index) in studentNotes" :key="index">{{ note }}</li>
```
Comment (m1073496) [inline src/components/StudentFeedback.vue:36]: I'd like to pick your brain over this directive and use of `:key` when we meet up later!

File: src/components/StudentFeedback.vue [hunk 8]
```diff
@@ -23,9 +23,38 @@
...
+        </ul>
+      </div>
+
+      <div class="form-container">
+        <button @click="toggleNoteForm" class="s-button-secondary show form-container__item">{{ showNoteForm ? 'Hide form' : 'Add new note' }}</button>
+        <form @submit.prevent="AddNote" v-if="showNoteForm" class="form-container__item">
```
Comment (m1073496) [inline src/components/StudentFeedback.vue:42]: Is this similar to `event.preventDefault()` ?

File: src/components/StudentFeedback.vue [hunk 9]
```diff
@@ -40,7 +69,12 @@ export default {
     return {
       projects: [],
       module: null,
```
Comment (m1073496) [inline src/components/StudentFeedback.vue:71]: Need to remove this. I don't think it is being used anymore.

**Review by m1073496 — APPROVED** [review]
Looks great, Nikki! I have no pressing issues. Consider it merged!

File: src/components/StudentFeedback.vue (hunk 8 above)
Comment (piknikki) [reply src/components/StudentFeedback.vue:42]: Yep!  same thing!

File: src/components/StudentFeedback.vue (hunk 7 above)
Comment (piknikki) [reply src/components/StudentFeedback.vue:36]: in this situation, key is not directive, v-bind is the directive. So it's saying, bind this key.  Just like with React, where it needs a key for unique things we create with mapping. We can chat later about it.

File: src/components/StudentFeedback.vue (hunk 9 above)
Comment (piknikki) [reply src/components/StudentFeedback.vue:71]: ok cool. maybe we can remove when we're resolving merge conflicts??

**Pull request merged**